test/evp_extra_test.c

```diff
@@ -581,6 +673,7 @@ int setup_tests(void)
 #ifndef OPENSSL_NO_EC
     ADD_TEST(test_EVP_PKCS82PKEY);
 #endif
+    ADD_ALL_TESTS(test_set_get_raw_keys, OSSL_NELEM(keys) * 2);
```

Comment:
This seems a little needlessly tricky.  Perhaps add a wrapper function that calls the current test_set_get_raw_keys with an additional pub/private param, something like this:
```
ADD_ALL_TESTS(test_get_set, OSSL_NELEM(keys));
...
int test_get_set(int tst)
{
    return test_set_get_raw_keys(tst, 1) && test_set_get_raw_keys(tst, 0)
}
```
I think this makes the existing test routine smaller and a bit more clear, too.
```